requests/signed_message_test.go

```diff
@@ -121,3 +143,15 @@ func ExampleSignedRequest_WithPemFile() {
 	fmt.Println(AuthKeyHeader(&msg.Key.PublicKey))
 	// Output: map[X-Yoti-Auth-Key:[MIGfMA0GCSqGSIb3DQEBAQUAA4GNADCBiQKBgQCpTiICtL+ujx8D0FquVWIaXg+ajJadN5hsTlGUXymiFAunSZjLjTsoGfSPz8PJm6pG9ax1Qb+R5UsSgTRTcpZTps2RLRWr5oPfD66bz4l38QXPSvfg5o+5kNxyCb8QANitF7Ht/DcpsGpL7anruHg/RgCLCBFRaGAodfuJCCM9zwIDAQAB]]
 }
+
+func TestSignedRequest_WithPemFile_NotPemEncodedShouldError(t *testing.T) {
+	msg := SignedRequest{}.WithPemFile([]byte("not pem encoded"))
+	assert.ErrorContains(t, msg.Error, "Not PEM-encoded")
+}
+
+func TestSignedRequest_WithPemFile_NotRSAKeyShouldError(t *testing.T) {
+	msg := SignedRequest{}.WithPemFile([]byte(`-----BEGIN RSA PUBLIC KEY-----
+` + exampleKey + `
+-----END RSA PUBLIC KEY-----`))
+	assert.ErrorContains(t, msg.Error, "Not an ESA Private Key")
```

Comment:
```suggestion
	assert.ErrorContains(t, msg.Error, "Not an RSA Private Key")
```

_(also need to change https://github.com/getyoti/yoti-go-sdk/blob/master/requests/signed_message.go#L96)_